### DISCUSSION TITLE
ssm parameter error

### DISCUSSION THREAD

**Discussion by Rajesh-2406** [learndevopsonline/learndevopsonline#1506] [Q&A]
![Screenshot 2023-10-05 111918](https://github.com/learndevopsonline/learndevopsonline/assets/136773140/c3d18a71-9ef9-4828-a8db-8200cd149ab3)
not acessing the parameter 
![Screenshot 2023-10-05 112121](https://github.com/learndevopsonline/learndevopsonline/assets/136773140/7fa6db29-ad91-40c1-b24f-e60da2f1206c)

**Comment by sreedharm07** [comment]
hi @Rajesh-2406 
can you  please check in aws UI  that iam policy  was created correctly ?

**Comment by Rajesh-2406** [comment]
![Screenshot 2023-10-05 130634](https://github.com/learndevopsonline/learndevopsonline/assets/136773140/f6cf5070-7892-4111-9dea-f11db42253e8)
created correctly

**Reply by sreedharm07** [reply]
can you please check wether the parameters are created in your aws parameters

**Reply by Rajesh-2406** [reply]
parameters also created

**Reply by Rajesh-2406** [reply]
![Screenshot 2023-10-05 132232](https://github.com/learndevopsonline/learndevopsonline/assets/136773140/4d9c6fb9-97ad-4884-baa6-db270858da53)

**Reply by sreedharm07** [reply]
Hi @Rajesh-2406 
i tried to re run your terraform code. got some errors 
here instead of policy arn you have given role arn please change it and please re run terraform code

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/9b84dd64-c1df-4b96-91fa-df0925f8b9d4)
